frontend/cypress/tests/zoomView/zoomView_hideCommitMessages.js

```diff
@@ -53,4 +53,174 @@ describe('hide all commit messages ', () => {
     cy.get('#tab-zoom .commit-message .body')
         .should('be.visible');
   });
+
+  it('should only display hide all commit messages when none are hidden', () => {
+    // open the commit panel
+    cy.get('.icon-button.fa-list-ul')
+        .should('be.visible')
+        .first()
+        .click();
+
+    cy.get('#tab-zoom .commit-message', { timeout: 90000 })
+        .should('be.visible');
+
+    // should only display 'hide all commit messages'
+    cy.get('#tab-zoom .toolbar--multiline')
+        .children()
+        .should('have.length', 1);
+
+    cy.get('#tab-zoom .toolbar--multiline > a')
+        .should('have.text', 'hide all commit messages');
```

Comment:
I think we should also check that the "show all commit messages" text is hidden. I think we might be able to do that by checking that only 1 item matches the selector.

Reply:
I think this should be addressed in the previous line(s)? 
```
    cy.get('#tab-zoom .toolbar--multiline')
        .children()
        .should('have.length', 1);
```
is checking that there's only one element, and the above is checking that the element has the correct text

Reply:
Right, yes. I was originally thinking of checking the text itself, then edited my comment to suggest about counting, which I didn't realize had already been done. Thanks!